packages/perseus-editor/src/widgets/__tests__/interactive-graph-editor-locked-figures.test.tsx

```diff
@@ -1202,6 +1202,13 @@ describe("InteractiveGraphEditor locked figures", () => {
                 lockedFigures: [defaultPolygon],
             });
 
+            // Close the "Start coordinates" section first to remove
+            // other accordions from the DOM.
+            const startCoordinatesHeading = screen.getByRole("button", {
+                name: "Start coordinates",
+            });
+            await userEvent.click(startCoordinatesHeading);
```

Comment:
I just noticed that the code doesn't quite match the comment here: we're clicking on the heading, which I assume might either expand or collapse the section depending on its current state.

A few ideas for fixing this:

- Could we assert that the section is expanded before clicking the heading?
- Could we assert that the section is collapsed after clicking the heading?
- Could we avoid the problem by testing a smaller component (i.e. only rendering the locked figures section)?

Reply:
Ah, I did it that way because it's always open by default. I can add an assertion to make it more clear though.

I think some smaller component testing is there in `locked-figures-section.test.tsx`. I remember doing it this way specifically to test it within the larger context of the editor page, because I wanted to test the editor's `onChange` function.